bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,244 @@
+| grid-reactive-current-l2              | Number:ElectricCurrent     | R          | Grid Reactive Current L2                 | Grid reactive current                                                                                |
+| grid-reactive-current-l3              | Number:ElectricCurrent     | R          | Grid Reactive Current L3                 | Grid reactive current                                                                                |
+| grid-active-current-l1                | Number:ElectricCurrent     | R          | Grid Active Current L1                   | Grid active current                                                                                  |
+| grid-active-current-l2                | Number:ElectricCurrent     | R          | Grid Active Current L2                   | Grid active current                                                                                  |
+| grid-active-current-l3                | Number:ElectricCurrent     | R          | Grid Active Current L3                   | Grid active current                                                                                  |
+| inverter-load-reactive-current-l1     | Number:ElectricCurrent     | R          | Inverter Load Reactive Current L1        |                                                                                                      |
+| inverter-load-reactive-current-l2     | Number:ElectricCurrent     | R          | Inverter Load Reactive Current L2        |                                                                                                      |
+| inverter-load-reactive-current-l3     | Number:ElectricCurrent     | R          | Inverter Load Reactive Current L3        |                                                                                                      |
+| inverter-load-active-current-l1       | Number:ElectricCurrent     | R          | Inverter Load Active Current L1          |                                                                                                      |
+| inverter-load-active-current-l2       | Number:ElectricCurrent     | R          | Inverter Load Active Current L2          |                                                                                                      |
+| inverter-load-active-current-l3       | Number:ElectricCurrent     | R          | Inverter Load Active Current L3          |                                                                                                      |
+| apparent-power                        | Number:Energy              | R          | Apparent Power                           | Apparent power                                                                                       |
```

Comment:
I think this channel type should be `Power` and not `Energy`. If i'm not mistaken i tis expressed in `Watts`.

Reply:
20240121
Changes done in README.md, thing-types.xml and FerroampChannelConfiguration.java